client/coral-admin/src/routes/Moderation/containers/Moderation.js

```diff
@@ -208,7 +212,7 @@ const withModQueueQuery = withQuery(gql`
     }) {
       ...CoralAdmin_Moderation_CommentConnection
     }
-    assets: assets {
+    assets: assets(query: {}) {
```

Comment:
Yeah either `assets {` or `assets: assets(query: null)` or `assets: assets(query: {limit: 0})` in the order of preference :-)